float64[] x_path
float64[] y_path
float64 yaw
---
float64 x
float64 y
---